core/services/balance_monitor_test.go

```diff
@@ -26,16 +23,17 @@ var nilBigInt *big.Int
 
 func TestBalanceMonitor_Start(t *testing.T) {
 	t.Run("updates balance from nil for multiple keys", func(t *testing.T) {
-		db := pgtest.NewGormDB(t)
-		ethKeyStore := cltest.NewKeyStore(t, db).Eth()
+		store, cleanup := cltest.NewStore(t)
+		defer cleanup()
+		ethKeyStore := cltest.NewKeyStore(t, store.DB).Eth()
```

Comment:
Please do not re-introduce store again. We're trying to remove our reliance on it.